makeTrainSet.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
makeTrainSet has now been moved to its own commit.

Extending the codebase to handle command line arguments should be a different ticket.